ACPO-model/src/train.py

```diff
@@ -328,10 +328,14 @@ def learn(args, train_data_path:str, test_data_path:str, log_dir:str, train_kwar
                 best_epoch = test_epoch_accuracy
 
                 if args.save_model:
-                    torch.save(model.state_dict(), os.path.join(log_dir, "plu.pt"))
-                    # bypass save_bp
-                    # save_pb(model, log_dir, num_features)
-                    save_pb_tmp_fuc(model, log_dir, num_features)
+                    print("Start saving models")
+                    if args.save_model_type == 'torch':
+                        scrip_model = torch.jit.script(model)
+                        scrip_model.save(os.path.join(log_dir, "modelfi.pt"))
```

Comment:
The file names here should not be hardcoded. Please fix.

Reply:
Thanks！Done.